automatic/samsung-magician/tools/chocolateyInstall.ahk

```diff
@@ -15,8 +15,16 @@ WinActivate
 ; Language selection
 ControlClick, OK, Select Setup Language,,,, NA
 
-; Welcome screen
-WinWait, %winTitleInstall%
+; Wait for either the overwrite confirmation dialog (for existing installs) or the Welcome screen (for new installs)
+WinWait, ahk_class i)#32770|TWizardForm ahk_exe Samsung_Magician_installer.tmp
```

Comment:
If I remember correctly, this breaks when a user has a different theme

Reply:
Are you referring to Windows' UI theme? I could not reproduce this with my work laptop (running Windows 10 Enterprise 20H2) or my normal dev environment.

(I could not test against the Chocolatey Testing Environment, since Windows locks this functionality behind activation)

Reply:
Yea, I think it was that. I am not sure though